odl/tomo/data/mrc2014.py

```diff
@@ -0,0 +1,197 @@
++---------+-------+---------+--------+-------------------------------+
+|20       |77-80  |Float32  |DMIN    |Minimum density value          |
++---------+-------+---------+--------+-------------------------------+
+|21       |81-84  |Float32  |DMAX    |Maximum density value          |
++---------+-------+---------+--------+-------------------------------+
+|22       |85-88  |Float32  |DMEAN   |Mean density value             |
++---------+-------+---------+--------+-------------------------------+
+|23       |89-92  |Int32    |ISPG    |Space group number 0, 1, or 401|
++---------+-------+---------+--------+-------------------------------+
+|24       |93-96  |Int32    |NSYMBT  |Number of bytes in extended    |
+|         |       |         |        |header                         |
++---------+-------+---------+--------+-------------------------------+
+|...      |       |         |        |                               |
++---------+-------+---------+--------+-------------------------------+
+|27       |105-108|Char     |EXTTYPE |Extended header type           |
++---------+-------+---------+--------+-------------------------------+
+|28       |109-112|Int32    |NVERSION|Format version identification  |
+|         |       |         |        |number                         |
++---------+-------+---------+--------+-------------------------------+
+|...      |       |         |        |                               |
++---------+-------+---------+--------+-------------------------------+
+|50-52    |197-208|Float32  |ORIGIN  |Origin in X, Y, Z used in      |
+|         |       |         |        |transform                      |
++---------+-------+---------+--------+-------------------------------+
+|53       |209-212|Char     |MAP     |Character string 'MAP' to      |
+|         |       |         |        |identify file type             |
++---------+-------+---------+--------+-------------------------------+
+|54       |213-216|Char     |MACHST  |Machine stamp                  |
++---------+-------+---------+--------+-------------------------------+
+|55       |217-220|Float32  |RMS     |RMS deviation of map from mean |
+|         |       |         |        |density                        |
++---------+-------+---------+--------+-------------------------------+
+"""
+# TODO: add nlabl stuff
+
+MRC_HEADER_BYTES = 1024
+MRC_SPEC_KEYS = {
+    'id': 'Long word',
+    'byte_range': 'Byte',
+    'dtype': 'Data type',
+    'name': 'Name',
+    'description': 'Description'}
+
+# Add more if needed
+MRC_DTYPE_TO_NPY_DTYPE = {
+    'Float32': np.dtype('float32'),
+    'Int32': np.dtype('int32'),
+    'Char': np.dtype('S1')}
+
+MRC_MODE_TO_NPY_DTYPE = {
+    0: np.dtype('uint8'),
+    1: np.dtype('int16'),
+    2: np.dtype('float32'),
+    6: np.dtype('uint16')}
+
+ANGSTROM = 1e-10
+
+
+class FileReaderMRC(FileReaderUncompressedBinary):
+
+    """Reader for the MRC file format(s).
+
+    By default, the MRC2014 format is used, see ``MRC_2014_SPEC`` for
```

Comment:
Hmm, you are correct. Irritating, not quite sure how to handle this. Is it not possible to add module variables to the doc?